### PR TITLE
iPad landscape UI

### PR DESCRIPTION
**Pull request by RyanRory**
Worked on the theming PR #672

### CODE REVIEW

File: Session/Closed Groups/NewClosedGroupVC.swift [hunk 1]
```diff
@@ -155,6 +159,11 @@ final class NewClosedGroupVC: BaseVC, UITableViewDataSource, UITableViewDelegate
         // Set up content
         setUpViewHierarchy()
     }
+    
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+        tableViewWidth?.constant = size.width
+    }
```
Comment (mpretty-cyro) [inline Session/Closed Groups/NewClosedGroupVC.swift:166]: We don't seem to actually set the `tableViewWidth` variable anywhere - can we just remove the changes to this file?

**Review by mpretty-cyro — CHANGES_REQUESTED** [review]
- [ ] When you haven't granted camera permission the 'Grant Camera Access' button is off screen (screenshot 1)
  - **Note:** This is only on the `NewDMVC` (it works fine on the communities and QR Code screens)

**(screenshot 1)**
Simulator Screen Shot - iPad Pro (9 7-inch) - 2022-10-10 at 17 07 55

File: SessionUIKit/Style Guide/Gradients.swift [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import UIKit
+
+@objc(LKGradient)
```
[the remark this answers is not in the document]
Comment (RyanRory) [reply SessionUIKit/Style Guide/Gradients.swift]: Nah, it must be added accidentally when I merged the other PRs. Will remove it.

File: Session/Closed Groups/NewClosedGroupVC.swift (hunk 1 above)
Comment (RyanRory) [reply Session/Closed Groups/NewClosedGroupVC.swift]: Removed the code, we don't need it anymore.

File: Session/Calls/CallVC.swift [hunk 3]
```diff
@@ -362,6 +363,17 @@ final class CallVC: UIViewController, VideoPreviewDelegate {
         NotificationCenter.default.addObserver(self, selector: #selector(audioRouteDidChange), name: AVAudioSession.routeChangeNotification, object: nil)
     }
     
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
+
+        if let existingSublayer = fadeView.layer.sublayers?[0] as? CAGradientLayer {
```
[the remark this answers is not in the document]
Comment (RyanRory) [reply Session/Calls/CallVC.swift]: We don't this anymore. I'll clean it up.

**Review by mpretty-cyro — CHANGES_REQUESTED** [review]

**Review by mpretty-cyro — APPROVED** [review]

**Comment by mpretty-cyro** [comment]
Closes #666

**Pull request merged**